pages/common/mdp.md

```diff
@@ -0,0 +1,19 @@
+# mdp
+
+> A command-line based tool to make presentations from markdown formatted files.
+
+- Start a presentation from a markdown file:
+
+`mdp {{presentation.md}}`
+
+- Disable fading transitions:
+
+`mdp --nofade {{presentation.md}}`
+
+- Invert colors for light background:
```

Comment:
I would expect such a tool to adapt to the current terminal's settings, so that if I had my terminal set to a light color, the presentations would appear in light background by default and `--invert` would show them in dark background. But if that's not the case, my suggestion would be something along the lines of: "Invert presentation colors to use a light background instead of the default dark background".